.github/ISSUE_TEMPLATE/engineering_spike.md

```diff
@@ -7,11 +7,40 @@ assignees: ''
 
```

Comment:
Above on line 5, the label is currently set with the `OCR` label. So ATM(im pretty sure) this will only apply to issues being made for the OCR board, we probably could just add the `dedupe` label and it should work for both.

Reply:
Might be best to replace whats in currently the `idwa_dedupe.yml` with this new one?